app/src/main/java/com/example/easysurvey/service/SurveyService.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.easysurvey.service
+
+import com.example.easysurvey.dao.IEasySurveyDAO
+import com.example.easysurvey.dto.Survey
+import com.example.easysurvey.ui.theme.RetrofitClientInstance
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.async
+import kotlinx.coroutines.withContext
+import retrofit2.awaitResponse
+
+interface IPlantService {
+    suspend fun fetchSurveys(): List<Survey>?
+}
+
```

Comment:
try to add an interface in but it was also giving me problem like with fetch so I copied like I did with fetch from the my plant diary as specified in class

Reply:
This class does not implement the interface.  Thus, the interface is not being used.